### DISCUSSION TITLE
Monthly vs Yearly Interest Rate

### DISCUSSION THREAD

**Discussion by TravisGriggsProjects** [MitchellGriggsTAFE/TafeCalc24#6] [Ideas]
I noticed the mortgage calculator template is presents the Yearly Interest Rate as 3.5 but the Monthly Interest Rate as 0.0029%. This appears incorrect to me, as the Yearly Interest Rate of 3.5 is already a percentage, so the proper calculation should simply be to divide by 12 for a Monthly Interest Rate of 0.29?

**Comment by MitchellGriggsTAFE** [comment]
That's a good pickup! I think the most important thing is to ensure consistency in the UI. That is, either both or neither of the Yearly/Monthly Interest Rate should have a percentage symbol attached. I leave the choice to you!
